code/API_definitions/geofencing-subscriptions.yaml

```diff
@@ -1536,12 +1624,38 @@ components:
               longitude: 7.10066
             radius: 2000
 
-    SUBSCRIPTION_ENDS:
-      description: The cloud event when a geofence subscription ends.
+    SUBSCRIPTION_STARTED:
+      description: The cloud event when a geofence subscription was started.
+      value:
+        id: "124003"
+        source: https://dataUsageServer.supertelco.com
+        type: org.camaraproject.geofencing-subscriptions.v0.subscription-started
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
```

Comment:
area is now required, so needs to be updated as well. device is optional so up to you